doc/cprover-manual/loop-invariant-synthesizer.md

```diff
@@ -0,0 +1,37 @@
+[CPROVER Manual TOC](../)
+
+## Loop Invariant Synthesis with goto-instrument
```

Comment:
```suggestion
## Loop Invariant Synthesis
```

Reply:
This is marked as resolved but hasn't actually been changed?